docs/deployment/registries.md

```diff
@@ -0,0 +1,124 @@
+# Image registries
+
+There are several places where you can store your docker image. In this section, we cover some of the most known ones.
+However, you can use any other of your preference.
+
+!!! warning "Build your image before hand"
+    In the following documentation we are assuming that you have already built your docker image.
+    You can find documentation on how to do it [here](./image.md).
+
+## DockerHub
+
+To use docker hub, follow Docker's [official documentation](https://docs.docker.com/docker-hub/repos/).
+
+## GitHub
+
+To store your image in GitHub you will need a repository. You can use your instance's one. For example:
+*https://github/yourusername/rdmrepo*. In addition, you will need to [create an access token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line). This access token must have
+`read:packages` and `write:packages` scopes.
+
+The first step is to login in GitHub's docker registry:
+
+``` console
+$ docker login docker.pkg.github.com -u <YOU_GITHUB_USERNAME> -p <YOUR_GITHUB_TOKEN>
+```
+
+Then, find your docker image and tag it with the url of your GitHub registry:
+
+```
+$ docker images
+REPOSITORY                                          TAG                 IMAGE ID            CREATED             SIZE
+demo-inveniordm                                     latest              9b6dd5ae6b48        17 hours ago        2.33GB
+
+$ docker tag demo-inveniordm docker.pkg.github.com/yourusername/rdmrepo/rdmimage:latest
+```
+
+!!! info "Project name"
+    Note that `rdmrepo` is the name of your GitHub repository. Therefore the tag is: ``docker.pkg.github.co/<your_github_user_name>/<your_github_repo_name>/<your_image_name>:<version>``
+
+Check that it was tagged correctly:
+
+``` console
+$ docker images                                                                   
+REPOSITORY                                                      TAG                 IMAGE ID            CREATED             SIZE
+demo-inveniordm                                                 latest              9b6dd5ae6b48        17 hours ago        2.33GB
+docker.pkg.github.com/yourusername/rdmrepo/rdmimage:latest      latest              9b6dd5ae6b48        17 hours ago        2.33GB
+```
+
+The last step is to push your package to the github registry:
+
+``` console
+docker push docker.pkg.github.com/yourusername/rdmrepo/rdmimage:latest
+```
+
+In order to use this image, you must set the following value in your `values.yaml` file (for both `web` and `worker`):
```

Comment:
`values.yaml` is for Openshift deployment only, right? What could be an indicator or a structure to make sure these things are clear to readers?

Reply:
It is not. The `values.yaml` file is a mere configuration yaml (as could be our `invenio.cfg` file) that helps you set config for this chart. In the end it just sets values that are replaced in the chart (jinja style) before deploying.

Reply:
Ah I see it's just kubernetes related.